recipe/meta.yaml

```diff
@@ -32,6 +32,18 @@ test:
   imports:
     - jupyter_packaging
 
+
+outputs:
+  - name: jupyter-packaging
+  - name: jupyter_packaging
```

Comment:
@conda-forge/jupyter-packaging lmk if there are any objections to exposing this package as 2 `conda`-installable packages, `jupyter-packaging` and `jupyter_packaging`

Reply:
Hey @timkpaine, I think having two packages would cause some confusion, why not stick with the same package name as pypi?

Reply:
Pypi supports dash and underscore as equivalent, so this actually makes the conda behavior the same as pypi

Reply:
Fair enough, I don't feel strongly either way

Reply:
no objections